identity-iota/Cargo.toml

```diff
@@ -48,7 +49,7 @@ features = ["blake2b"]
 
 [dev-dependencies]
 proptest = { version = "1.0.0", default-features = false, features = ["std"] }
-tokio = { version = "1.15", default-features = false, features = ["macros"] }
+tokio = { version = "1.17.0", default-features = false, features = ["macros"] }
```

Comment:
I'm unsure of the circumstances in which we should not update to the latest version of Rust dependencies, particularly when they are semver compatible.

In this case I found it prudent specifically because [`tokio 1.17.0` contains fixes to avoid panics](https://github.com/tokio-rs/tokio/blob/master/tokio/CHANGELOG.md#1170-february-16-2022).

I'll keep it in mind that we may not want to update all dependencies for compatibility reasons, specifically if those updates contain API-breaking changes.

Reply:
I have no such justification for `strum`, however. I just wanted the latest version (and forgot to update it everywhere).